opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/user-test.js

```diff
@@ -684,3 +699,181 @@ describe('User has no capability query behavior', () => {
     });
   });
 });
+
+describe('User has no settings capability and is organization admin query behavior', () => {
+  let userInternalId;
+  let userEditorId;
+  let testOrganizationId;
+  let amberGroupId;
+  /*  beforeAll(async () => {
+    // Need to add granted_groups to TEST_ORGANIZATION because of line 533 in domain/user.js
+    amberGroupId = await getGroupIdByName(AMBER_GROUP.name);
+
+    const UPDATE_QUERY = gql`
+      mutation OrganizationEdit($id: ID!, $input: [EditInput]!) {
+        organizationFieldPatch(id: $id, input: $input) {
+          id
+          name
+          grantable_groups {
+            id
+          }
+        }
+      }
+    `;
+    const queryResult = await adminQuery({
+      query: UPDATE_QUERY,
+      variables: { id: testOrganizationId, input: { key: 'grantable_groups', value: [amberGroupId] } },
+    });
+    expect(queryResult.data.organizationFieldPatch.grantable_groups.length).toEqual(1);
+    expect(queryResult.data.organizationFieldPatch.grantable_groups[0]).toEqual({ id: amberGroupId });
+  }); */
+  afterAll(async () => {
+    it('should remove the capability to administrate the Organization', async () => {
```

Comment:
Tests inside afterAll function? Thats seems odd

Reply:
it's an essential clean up step, but it's still clean up, which is why I've put it in a beforeAll.
I had so many problems with these tests that I added assertions to make sure the steps were all done correctly...In your opinion, do I keep a beforeAll without “it” or do I keep the “it” that I move after the others?

Reply:
I suppose you mean `afterAll` and not `beforeAll`. 
If it's clean up then you are not really testing stuff so keep `afterAll` but without any `it` or assertions in it. If it's indeed a test then it should be done outside the `afterAll` step